test/libyul/yulOptimizerTests/redundantAssignEliminator/for_continue_3.yul

```diff
@@ -0,0 +1,31 @@
+{
+	// Cannot be removed, because we might run the loop only once
+	let x := 1
```

Comment:
Interesting - this could actually be removed.

Reply:
Ah, but it is not an assignment! Can you turn it into a declaration plus assignment?